src/vue/pages/sales/Sales.vue

```diff
@@ -0,0 +1,200 @@
+<template>
+  <div class="all-sales">
+    <div class="sales__state-filter">
+      <select-field
+        :is-value-translatable="true"
+        :disabled="!isLoaded"
+        v-model="filters.state"
+        :values="Object.values(SALE_STATES)"
+        key-as-value-text="labelTranslationId"
+        class="sales-asset-selector__field app__select app__select--no-border"
+      />
+    </div>
+    <template v-if="filteredSales.length">
+      <div class="sales__sale-cards">
+        <drawer :is-shown.sync="isDetailsDrawerShown">
+          <template slot="heading">
+            {{ 'sales.overview-title' | globalize }}
+          </template>
+          <sale-overview :sale="selectedSale" />
+        </drawer>
+
+        <sale-card
+          class="sales__sale-card"
+          v-for="sale in filteredSales"
+          :key="sale.id"
+          :sale="sale"
+        />
+      </div>
+    </template>
+
+    <template v-else-if="isLoaded">
+      <no-data-message
+        icon-name="inbox"
+        :title="'sales.no-sales-title' | globalize"
+        :message="'sales.no-sales-desc' | globalize"
+      />
+    </template>
+
+    <template v-else>
+      <loader :message-id="'sales.loading-msg'" />
+    </template>
+
+    <!--
+    v-show is a hack to hide `More` button if there are no sales,
+    matching the filtering criteria (when no data message is shown).
+  -->
+    <collection-loader
+      v-show="filteredSales.length"
+      class="sales__loader"
+      :first-page-loader="recordsLoader"
+      @first-page-load="setRecords"
+      @next-page-load="extendRecords"
+    />
+  </div>
+</template>
+
+<script>
+import config from '@/config'
+
+import Drawer from '@/vue/common/Drawer'
+import Loader from '@/vue/common/Loader'
+import CollectionLoader from '@/vue/common/CollectionLoader'
+import NoDataMessage from '@/vue/common/NoDataMessage'
+import SelectField from '@/vue/fields/SelectField'
+
+import SaleOverview from '@/vue/pages/sales/SaleOverview'
+import SaleCard from '@/vue/pages/sales/SaleCard'
+
+import { Sdk } from '@/sdk'
+import { vueRoutes } from '@/vue-router/routes'
+import { mapGetters } from 'vuex'
+import { vuexTypes } from '@/vuex'
+
+import { SaleRecord } from '@/js/records/entities/sale.record'
+
+const SALE_STATES = {
+  live: {
+    labelTranslationId: 'sales.sale-live-state',
+    value: 'live',
+  },
+  upcoming: {
+    labelTranslationId: 'sales.sale-upcoming-state',
+    value: 'upcoming',
+  },
+  all: {
+    labelTranslationId: 'sales.sale-all-state',
+    value: 'all',
+  },
+}
+
+export default {
+  name: 'all-oprtunities',
```

Comment:
Typo. `all-sales`. Also, you need to name this file correct. `sales/Sales.vue` is shitty